install/kubernetes/helm/istio/values.yaml

```diff
@@ -1,4 +1,85 @@
-# Common settings.
+ #
+# Gateways Configuration, refer to the charts/gateways/values.yaml 
+# for detailed configuration
+#
+gateways:
+  enabled: true
```

Comment:
how does this interact with the values.yaml in gateway/ >

Reply:
this yaml will overwrite subchart's value.  Hope this addresses your other concern.

Reply:
@rshriram order of precedence for Helm K/V pairs:

1. with either CLI actions of `--set` or `'f`, the last value set for a key is used.  E.g. `--set key=a key=b` key will always be b.
1. top level chart (helm/istio) values.yaml
1. sub chart values.yaml